src/client/components/CSRFForm.tsx

```diff
@@ -24,4 +24,6 @@ const CSRFForm = forwardRef<HTMLFormElement, CSRFFormProps>(({ children, method,
   );
 });
 
+CSRFForm.displayName = 'CSRFForm';
```

Comment:
Lint fix, giving the component a name. Nice when using react dev tools to look at the component tree